client/my-sites/checkout/checkout-thank-you/marketplace/use-themes-thank-you-data.tsx

```diff
@@ -90,12 +84,35 @@ export function useThemesThankYouData( themeSlugs: string[] ): ThankYouData {
 	);
 	const isAtomicNeeded = hasDotOrgThemes || hasExternallyManagedThemes;
 
+	// theme will automatically activate if user has a plan that includes it
+	// the CTA is conditional to whether a theme activates automatically or not
+	const hasThemeIncludedInCurrentPlan = useSelector( ( state ) =>
+		allThemesFetched
+			? themesList.map( ( theme ) => siteId && canUseTheme( state, siteId, theme.id ) )
```

Comment:
```suggestion
			? themesList.some( ( theme ) => siteId && canUseTheme( state, siteId, theme.id ) )
```

Should we use some here?